paddle/fluid/inference/tests/api/tester_helper.h

```diff
@@ -760,15 +764,16 @@ void CompareAccuracy(
     const std::vector<std::vector<PaddleTensor>> &output_slots_quant,
     const std::vector<std::vector<PaddleTensor>> &output_slots_ref,
     int compared_idx) {
-  if ((FLAGS_enable_fp32 && (FLAGS_enable_int8 || FLAGS_enable_bf16)) &&
+  if ((FLAGS_enable_fp32 &&
+       (FLAGS_enable_int8_ptq || FLAGS_enable_int8_qat || FLAGS_enable_bf16)) &&
       (output_slots_quant.size() == 0 || output_slots_ref.size()) == 0)
     throw std::invalid_argument(
         "CompareAccuracy: output_slots vector is empty.");
```

Comment:
This if could use some explanation. It seems counter intuitive that we check ((fp32 && non_fp32) &&  some output_slots vector).